### PR TITLE
[RELEASE] 0.18.16

### PR DESCRIPTION
**Pull request by Kilo59**
release prep for 0.18.16

### CODE REVIEW

File: docs/docusaurus/docs/changelog.md [hunk 1]
```diff
@@ -10,6 +10,21 @@ title: Changelog
 - Deprecation warnings are accompanied by a moniker (as a code comment) indicating when they were deprecated.  For example: `# deprecated-v0.13`
 - Changes to methods and parameters due to deprecation are also noted in the relevant docstrings.
 
+### 0.18.16
+* [FEATURE] SnowflakeDatasource - Require `database` and `schema` fields ([#9980](https://github.com/great-expectations/great_expectations/pull/9980))
+* [FEATURE] Snowflake - use `Datasource` level `schema` when creating `TableAsset` ([#9992](https://github.com/great-expectations/great_expectations/pull/9992))
```
Comment (Kilo59) [inline docs/docusaurus/docs/changelog.md]: Both of these changes were reverted in a subsequent PR.
Should we even include this in the changelog.

My instinct is to say remove them (or use a ~strikethrough~) ... but there are some small QOL updates in these PR's that were retained even if the titular "features" were reverted. 🤷

Comment (billdirks) [reply docs/docusaurus/docs/changelog.md]: I would remove these and the revert PRs from the changelog. Or prefix `REVERTED -` to the description.

**Review by billdirks — APPROVED** [review]
LGTM.

File: docs/docusaurus/docs/changelog.md (hunk 1 above)
Comment (Kilo59) [inline docs/docusaurus/docs/changelog.md]:
```suggestion
* REVERTED -[FEATURE] SnowflakeDatasource - Require `database` and `schema` fields ([#9980](https://github.com/great-expectations/great_expectations/pull/9980))
* REVERTED - [FEATURE] Snowflake - use `Datasource` level `schema` when creating `TableAsset` ([#9992](https://github.com/great-expectations/great_expectations/pull/9992))
```

File: docs/docusaurus/docs/changelog.md [hunk 2]
```diff
@@ -10,6 +10,21 @@ title: Changelog
...
+* [FEATURE] Add new `ConfigUri` type ([#10000](https://github.com/great-expectations/great_expectations/pull/10000))
+* [FEATURE] Restrict substitutable sections for `Snowflake.connection_string` ([#10002](https://github.com/great-expectations/great_expectations/pull/10002))
```
Comment (Kilo59) [inline docs/docusaurus/docs/changelog.md]:
```suggestion
* REVERTED - [FEATURE] Restrict substitutable sections for `Snowflake.connection_string` ([#10002](https://github.com/great-expectations/great_expectations/pull/10002))
```

**Pull request merged**